code/src/cljs/sixsq/nuvla/ui/edges/views_utils.cljs

```diff
@@ -63,7 +67,11 @@
                                      :corner    true
                                      :color     "blue"}])]
                         (or name id)]
-          :meta        (str (@tr [:created]) " " (-> created time/parse-iso8601 time/ago))
+          :meta        [:<>
+                        [:div (str (@tr [:created]) " " (date-string->time-ago created))]
+                        (when next-heartbeat [:div  (str (@tr [:online])
+                                                         " "
+                                                         (date-string->time-ago next-heartbeat))])]
```

Comment:
This is the calculated next-heartbeat. The value of last online is calculated in the NuvlaEdge details page. We should unify both in a function so that we don't show different values.